flang/test/Semantics/OpenMP/cancellation-construct-type.f90

```diff
@@ -0,0 +1,11 @@
+!RUN: %python %S/../test_errors.py %s %flang_fc1 %openmp_flags
+
+subroutine f(x)
+  integer :: x
+!ERROR: Cancellation construct type is not allowed on SECTIONS
```

Comment:
I think the meaning of this error will not be obvious to the user. Perhaps "PARALLEL cannot follow SECTIONS"?